erpnext/stock/doctype/batch/batch.py

```diff
@@ -93,9 +93,7 @@ def autoname(self):
 				['create_new_batch', 'batch_number_series'])
 
 			if create_new_batch:
-				if self.batch_naming_series:
-					self.batch_id = make_autoname(self.batch_naming_series)
-				elif batch_number_series:
+				if batch_number_series:
```

Comment:
@meshramsaroj you have to fix test because of this change. do not remove code from controller and doctype to fixed test